tests/ui/specs/VirtualList/VirtualGridList/ElementSize/VirtualGridList-GT-28446-specs.js

```diff
@@ -0,0 +1,62 @@
+const Page = require('../VirtualGridListPage');
+
+describe('Change Sapcing', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should change spotlight size when spaceing change [GT-28446]', function () {
+		// Check default item size.
+		const defaultItemWidth = Page.getItemSize().width;
+		const defaultItemHeight = Page.getItemSize().height;
+		const defaultSpacingWidth = Page.itemSpacing(0, 1).width;
+		const defaultSpacingHeight = Page.itemSpacing(0, 5).height;
+		Page.item(0).moveTo();
+		Page.spotlightSelect();
+		const defaultSpotlightWidth = Page.spotlightSize().width;
+		const defaultSpotlightHeight = Page.spotlightSize().height;
+		// Step 3: Knobs > VirtualGridList > spacing > 200
```

Comment:
Please add some empty line properly. I'm out of breath to read.

Reply:
Suggested change.